docs/README.md

```diff
@@ -39,7 +39,17 @@ If you see a project that is under-maintained, submit PRs to fix it as if it was
 
 If you need to contact the nix-community admins, they are listed [here](https://github.com/nix-community/infra#nix-community-administrators).
 
-Come chat with us in [#nix-community](https://matrix.to/#/#nix-community:nixos.org) using the matrix protocol!
+## Support
+
+For casual chat, come join us in the [nix-community](https://matrix.to/#/#nix-community:nixos.org) room.
+
+For urgent issues, ping us in the Matrix room (see the admin list below).
```

Comment:
To keep these sections the same probably easier to move support above admins.

Reply:
thanks